### ISSUE TITLE
Replace panels with breakout task forces

### ISSUE THREAD

**Issue by elf-pavlik** [solid/process#324]
Having participated in numerous Solid Panels since 2019 I think it is time for a retrospective and reorganization.

### 2019 - 2022

Panels were most active, incubating proposals and advancing them into dedicated repositories.
There was a special case of the Editors Team which I understood was an invite-only meeting.

### 2022 - 2023

Many panels stop their activity:

* Authentication Panel - practically no activity
* Authorization Panel - practically no activity
* Data Interoperability Panel - some irregular activity
* Notifications - very active in 2022, falling off in 2023

At the same time, there is a varying degree of work happening on the dedicated repositories and implementations!

### 2023 - 

I think we should anticipate that Solid WG will be established and will be organized independently of CG while keeping coordinated. Based on CG charter and conversations around deliverables (protocol + dependencies) solid/solid-wg-charter#37 I would expect it to cover the following areas:

* Solid Storage
* Authorization and Authentication
* Notifications (maybe)

There are still various areas, like discovery (which can depend on AuthZ) and other app interop-related concerns which the CG should continue its work on. I think the following might be one of the simpler and more effective approaches for the CG:

* Single regular weekly meeting Solid-CG, which we should consider extending to 2 hours and always have open PR or Issue (at least gh Discussion) for each topic on the agenda.
* If a certain area regularly starts taking up a significant part of that meeting, and we regularly need to push topics to next week, we should call a breakout Task Force which has additional meetings and to process the workload.
* We don't create a repo for the Task Forces, only specifications have their repositories, and CG with TF work on those spec-focused repositories.
* Instead of having incubating documents in the panel repo and creating new repo later on, the initial draft gets created in the personal repository of the person who is proposing it, they can always invite collaborators! As soon as the proposal gets accepted as CG draft it gets transferred to @solid github org.

--------

I think we should do some retrospective first and after that discuss what I propose and what others will be interested in proposing. BTW [Solid Panels](https://github.com/solid/process/blob/main/panels.md) document is pretty outdated, it links to a Zoom call and W3C wiki for minutes.

**Comment by elf-pavlik** [comment]
Let's focus on
* #323 

As I understand now it should include addressing the above.

**Comment by elf-pavlik** [comment]
Reopening since #323 is practically done

**Comment by elf-pavlik** [comment]
@woutermont created a set of PRs to update panel's meeting information:

* https://github.com/solid/authentication-panel/pull/238
* https://github.com/solid/authorization-panel/pull/330
* https://github.com/solid/data-interoperability-panel/pull/318
* https://github.com/solid/solid-oidc/pull/223
* https://github.com/solid/webid-profile/pull/106
* https://github.com/solid/notifications-panel/pull/102

**Comment by woutermont** [comment]
Context: awaiting confirmation of the idea to obsolete panels, a number of people (possible popping their heads up after summer slumber) were confused when no one turned up in some panel meetings that were still scheduled in the agenda.

**Comment by csarven** [comment]
Right. So, this is aligned with the initial suggestion in https://github.com/solid/specification/pull/458#issue-1390870735 to transition from panels to task forces. We needed to iron out a bunch of stuff along the way. Here we are.

I've updated the status of data-interoperability-panel, test-suite-panel, notifications-panel, webid-profile in the CG calendar to "tentative" for the time being.

BTW, webid-profile isn't and wasn't intended to be a panel. It is intended as the first task force (before literally calling it as such).

**Comment by csarven** [comment]
PRs references in https://github.com/solid/process/issues/324#issuecomment-1719720918 are now resolved.

Pavlik, everyone:

I suggest closing this issue. Solid Process needs to be updated any way. Discussion pertaining to CG task force can carry out in solid/specification.
